themes/docsy/layouts/partials/section-index.html

```diff
@@ -16,7 +16,7 @@
     {{/* Otherwise we show a nice formatted list of subpages with page descriptions */}}
     <hr class="panel-line">
         {{ range $pages }}
-            {{ if eq .Parent $parent }}
+            {{ if and (eq .Parent $parent) (ne .Params.page_hide true) }}
```

Comment:
Revert this change. We should not change the files in the `/themes` directory, as any changes here will be overridden next time we update our version of Docsy.